packages/node/src/indexer/sandbox.ts

```diff
@@ -51,7 +51,7 @@ export class IndexerSandbox extends NodeVM {
   private config: NodeConfig;
   entry: string;
 
-  constructor(option: SandboxOption, config?: NodeConfig) {
+  constructor(option: SandboxOption, config: NodeConfig) {
```

Comment:
pls also change all `this.config?.logLevel` in this file to `this.config.logLevel`